docs/components/HTTP.md

```diff
@@ -161,6 +161,20 @@ func NewCompressionMiddleware(deflateLevel int, ignoreRoutes ...string) Middlewa
 
 ```
 
+### Error Logging
+
+It is possible to configure specific status codes that, if returned by an HTTP handler, the response's error will be logged.
+
+This configuration must be done using the `PATRON_HTTP_STATUS_ERROR_LOGGING` environment variable. The syntax of this variable is based on PostgreSQL syntax and allows providing ranges.
+
+For example, setting this environment variable to `409;[500,600)` that an error will be logged if an HTTP handler returns either:
+* A status code 409
+* A status code greater or equal than 500 (the bracket represents the inclusion) and strictly smaller than 600 (the parenthesis represents the exclusion)
+
+Be it a specific status code or a range; each element must be delimited with `;`.
+
+To enable error logging, we must add the `NewLoggingTracingMiddleware` middleware.
```

Comment:
The last sentence seems unnecessary since it will be enabled if the env var is there. It will also confuse user as to if they have to add the middleware or not. Should we remove it?

Reply:
> since it will be enabled if the env var is there
Is it? If we don't add the middleware created in `NewLoggingTracingMiddleware`, we are not going to log errors even though the env var is set.

Reply:
`NewLoggingTracingMiddleware` is automatically added to routes that we mark as traced. So from the user's perspective they have just to enable it correct. 

```
if rb.trace {
		middlewares = append(middlewares, NewLoggingTracingMiddleware(rb.path))
	}
```
in route.go.
Do i miss something?
Maybe we should also mention this in the doc.